apps/dotcom-worker/src/lib/utils/createPersistQueue.test.ts

```diff
@@ -0,0 +1,58 @@
+import { createPersistQueue } from './createPersistQueue'
+
+const tick = () => Promise.resolve()
+
+const resolvable = () => {
```

Comment:
`import {promiseWithResolve} from '@tldraw/utils'`

not identical, but it's a `Promise` with `resolve` and `reject` methods